cmd/ship-it-ecrconsumer/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"ship-it/internal/ecrconsumer"
+	"ship-it/internal/ecrconsumer/config"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/metrics/dogstatsd"
+)
+
+func main() {
```

Comment:
Lots of duplication between `main`s, though it's all boilerplate service init. We could consider abstracting this later.